docs/reqs/reqs.tex

```diff
@@ -192,118 +192,130 @@ \section{Goal}
 \end{enumerate}
 
 \subsection{Computing Platform}
-\begin{enumerate}[{F.CP}.1:]
+\begin{enumerate}[label=F.CP.\arabic*, wide=1cm, widest=3cm, leftmargin=*, font=\bfseries, noitemsep,topsep=0pt, parsep=4pt, partopsep=0pt]
 	\item Captures video at a minimum resolution of 640x480 at 10fps.
 	\item Onboard FPGA is capable of performing parallel matrix computation to speedup machine learning.
 \end{enumerate}
 
 \subsection{Machine Learning Implementation}
-\begin{enumerate}[{F.ML}.1:]
-\item A machine learning model, implemented on the FPGA, is capable of detecting pedestrians from image frames. captured using the camera.
-\item The machine learning model's processing latency is one second or less.
-
+\begin{enumerate}[label=F.ML.\arabic*, wide=1cm, widest=3cm, leftmargin=*, font=\bfseries, noitemsep,topsep=0pt, parsep=4pt, partopsep=0pt]
+	\item A machine learning model, implemented on the FPGA, is capable of detecting pedestrians from image frames. captured using the camera.
+	\item The machine learning model's processing latency is one second or less.
 \end{enumerate}
 
 \subsubsection{Base Station}
-\begin{enumerate}[{F.BS}.1:]
+\begin{enumerate}[label=F.BS.\arabic*, wide=1cm, widest=3cm, leftmargin=*, font=\bfseries, noitemsep,topsep=0pt, parsep=4pt, partopsep=0pt]
     \item Ground station displays the video data overlaid with the ML model data.
 	\item ML model data is presented in the form of bounding boxes.
 \end{enumerate}
 
 \section{Non-Functional Requirements}\label{section:nonfuncrec}
 
-TODO: We might have to split non-functional requirements into nf-reqs and constraints (the capstone docs say they're separate, but PC's don't).
-
 \subsection{Multirotor RPAS}
-\begin{enumerate}[{NF.FP}.1:]
-    \item Drone has at least 50\% flight duration of that without the payload --- 10 minutes minimum.
-    \item The drone must be able to fly in fair conditions (assuming no rain, heavy winds, etc.).
-    \item The drone should be capable of auto-leveling, drift-correction, and perform emergency landing.
-    \item The drone should have systems for low-battery alarm and emergency protocol for low-battery and loss of control radio.
-\end{enumerate}
-
-\subsection{System Reliability and Durability}
-\begin{enumerate}[{NF.SR}.1:]
-    \item Brushless DC Motors used on the drone should have a life span of 10,000 flight-hours.
-    \item Non-computation-platform electronics are expected to have a life span of 20,000 flight hours if the procedures are followed correctly
-    \item The lithium polymer battery for the drone is expected to retain 75\% of its charge after 200 charge/discharge cycles. (TODO: make citation \url{http://learningrc.com/lipo-battery/}) 
+\begin{enumerate}[label=NF.DR.\arabic*, wide=1cm, widest=3cm, leftmargin=*, font=\bfseries, noitemsep,topsep=0pt, parsep=4pt, partopsep=0pt]
+	\item The multirotor RPAS shall be able to carry a maximum payload mass of 2.0 kg or 4.4 lb and achieve flight.
```

Comment:
Stick to metric here, no point in citing both

Reply:
Ok, I’ll correct it when I get on the computer tomorrow